### PR TITLE
Audio Quick Start

### PR DESCRIPTION
**Pull request by ericgiannini**

### CODE REVIEW

**Comment by TheChrisVela** [comment]
Awesome!

File: docs/enable-audio.md [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+# Uncomment the next line to define a global platform for your project
+platform :ios, '9.0'
+
+source "https://github.com/Nexmo/PodSpec.git"
+source 'git@github.com:CocoaPods/Specs.git'
+
+target 'AudioQuickStart' do
+  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
+  use_frameworks!
+  pod "NexmoConversation", :git => "https://github.com/nexmo/conversation-ios-sdk.git", :branch => "master" #latest release
```
Comment (shams-ahmed) [inline docs/enable-audio.md:53]: just `pod "NexmoConversation"` fine

File: docs/enable-audio.md [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+
+Since we will be tapping into protected device functionality we will have to ask for permission. We will update our `.plist` as well as display an alert. After permissions we will add AVFoundation class, set up audio from within the SDK and add a speaker emoji for our UI 🔈
+
+## 2.1 Xcode Permission
+
+Open up the raw version of the `.plist`. Drop the following lines of code in there.
+
+```
+<key>NSMicrophoneUsageDescription</key>
+	<string>audio call permission</string>
```
Comment (shams-ahmed) [inline docs/enable-audio.md:260]: add a pic?

File: docs/enable-audio.md [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+
+  }
+
+```
+
+Configure the text property on the button's text label to display either speaker 🔈 for enabled audio or else mute 🔇 for disabled audio.
+
+## 2.5 Console logs
+
+By implementing our enable / disable functions, we will see the updates right there inside of Xcode in the console log.
```
Comment (shams-ahmed) [inline docs/enable-audio.md:337]: filter via `RTC: xxx`

**Pull request merged**